cmd/lakectl/cmd/branch.go

```diff
@@ -43,15 +43,15 @@ var branchListCmd = &cobra.Command{
 
 		rows := make([][]interface{}, len(response))
 		for i, row := range response {
-			rows[i] = []interface{}{row}
+			rows[i] = []interface{}{swag.StringValue(row.ID), swag.StringValue(row.CommitID)}
 		}
 
 		ctx := struct {
 			BranchTable *Table
 			Pagination  *Pagination
 		}{
 			BranchTable: &Table{
-				Headers: []interface{}{"Branch"},
+				Headers: []interface{}{"Branch", "Commit ID"},
 				Rows:    rows,
 			},
```

Comment:
why did we add commit ID to branch log?

Reply:
did some digging - https://github.com/treeverse/lakeFS/commit/548e1d1d8f19f6ad05390fd93d610a773708b603#diff-d57f018b31eac5e4a0a201b50e3f2f5d0c193e682719b5ebb65bb612500991e7R56